protocol/messenger_handler.go

```diff
@@ -3683,3 +3682,7 @@ func (m *Messenger) addNewKeypairAddedOnPairedDeviceACNotification(keyUID string
 	}
 	return nil
 }
+
+func (m *Messenger) HandleRekeyGroup(state *ReceivedMessageState, message *protobuf.CommunityMessageArchiveMagnetlink, statusMessage *v1protocol.StatusMessage) error {
+	return nil
```

Comment:
Missing implementation?

Reply:
it's a left over, I didn't create a new application method after all, but did at some point, I will remove it